<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_menu_ui</class>
 <widget class="QDialog" name="gestion_menu_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/photos/b.png);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>gerer menu</string>
    </attribute>
    <widget class="QTabWidget" name="jour">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>781</width>
       <height>521</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="ajout">
      <attribute name="title">
       <string>ajouter  menu</string>
      </attribute>
      <widget class="QLineEdit" name="designation_menu">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>20</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 3px;
border-top-color: rgb(255, 85, 0);

border-bottom-color: rgb(0, 170, 0);
border-right-color: rgb(255, 255, 0);
border-left-color: rgb(255, 255, 0);
</string>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ff5500;&quot;&gt;Designation Menu&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ajout_menu">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>240</y>
         <width>161</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 3px;
border-color: red;

</string>
       </property>
       <property name="text">
        <string>Creer menu </string>
       </property>
      </widget>
      <widget class="QLabel" name="l_controle_saisie_2">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>130</y>
         <width>231</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>130</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ff5500;&quot;&gt;JOUR&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>80</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ff5500;&quot;&gt;Preparateur&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="preparateur_menu">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>70</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 3px;
border-top-color: rgb(255, 85, 0);

border-bottom-color: rgb(0, 170, 0);
border-right-color: rgb(255, 255, 0);
border-left-color: rgb(255, 255, 0);
</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="jour_menu">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>120</y>
         <width>261</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 3px;
border-top-color: rgb(255, 85, 0);

border-bottom-color: rgb(0, 170, 0);
border-right-color: rgb(255, 255, 0);
border-left-color: rgb(255, 255, 0);
</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>190</y>
         <width>261</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#aa0000;&quot;&gt;Veuillez composer votre menu &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>340</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ff5500;&quot;&gt;Plat principal&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>340</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ff5500;&quot;&gt;salade&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>340</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ff5500;&quot;&gt;dessert&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_1">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>281</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>281</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_3">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>281</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="ma">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>390</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="afficher">
      <attribute name="title">
       <string>afficher la liste des menus</string>
      </attribute>
      <widget class="QTableView" name="affich">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>110</y>
         <width>701</width>
         <height>241</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="recherche_afficher">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>30</y>
         <width>291</width>
         <height>61</height>
        </rect>
       </property>
      </widget>
      <widget class="QCommandLinkButton" name="commandLinkButton">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>30</y>
         <width>81</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>10</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;

</string>
       </property>
       <property name="text">
        <string>Recherche</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../trouver-recherche-zoom-icone-4701-48.png</normaloff>../../../trouver-recherche-zoom-icone-4701-48.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="excel">
       <property name="geometry">
        <rect>
         <x>1050</x>
         <y>10</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>excel</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>suppression</string>
      </attribute>
      <widget class="QTableView" name="supprimermenu">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>681</width>
         <height>211</height>
        </rect>
       </property>
      </widget>
      <widget class="QCommandLinkButton" name="commandLinkButton_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>181</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>rechercher par nom</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../trouver-recherche-zoom-icone-4701-48.png</normaloff>../../../trouver-recherche-zoom-icone-4701-48.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>300</y>
         <width>201</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>le menur à supprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimer_22">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>310</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 4px;
border-color: red;</string>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom_22">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>310</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 3px;
border-top-color: rgb(255, 85, 0);

border-bottom-color: rgb(0, 170, 0);
border-right-color: rgb(255, 255, 0);
border-left-color: rgb(255, 255, 0);
</string>
       </property>
       <property name="placeholderText">
        <string>entrer le menu que vous voulez supprimer</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="recherche_afficher_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>10</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>modifier</string>
      </attribute>
      <widget class="QTableView" name="modifier_3">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>50</y>
         <width>501</width>
         <height>281</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_32">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>id</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_33">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>entree</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_34">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>designation</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="designation">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 3px;
border-top-color: rgb(255, 85, 0);

border-bottom-color: rgb(0, 170, 0);
border-right-color: rgb(255, 255, 0);
border-left-color: rgb(255, 255, 0);
</string>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="jour_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>160</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 3px;
border-top-color: rgb(255, 85, 0);

border-bottom-color: rgb(0, 170, 0);
border-right-color: rgb(255, 255, 0);
border-left-color: rgb(255, 255, 0);

</string>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="id_menu">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
border-style: solid;
border-width: 3px;
border-top-color: rgb(255, 85, 0);

border-bottom-color: rgb(0, 170, 0);
border-right-color: rgb(255, 255, 0);
border-left-color: rgb(255, 255, 0);
</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="p1">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>210</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 3px;
border-top-color: rgb(255, 85, 0);

border-bottom-color: rgb(0, 170, 0);
border-right-color: rgb(255, 255, 0);
border-left-color: rgb(255, 255, 0);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
      <widget class="QCommandLinkButton" name="commandLinkButton_6">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>0</y>
         <width>181</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>rechercher par nom</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../trouver-recherche-zoom-icone-4701-48.png</normaloff>../../../trouver-recherche-zoom-icone-4701-48.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="label_35">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>jour</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_36">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>270</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>plat</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="p2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>270</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 3px;
border-top-color: rgb(255, 85, 0);

border-bottom-color: rgb(0, 170, 0);
border-right-color: rgb(255, 255, 0);
border-left-color: rgb(255, 255, 0);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="preparateur">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>110</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 3px;
border-top-color: rgb(255, 85, 0);

border-bottom-color: rgb(0, 170, 0);
border-right-color: rgb(255, 255, 0);
border-left-color: rgb(255, 255, 0);
</string>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier_4">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>380</y>
         <width>131</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 4px;
border-color: red;

</string>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="recherche_afficher_6">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>10</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_37">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>preparateur</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>350</y>
         <width>161</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 4px;
border-color: red;</string>
       </property>
       <property name="text">
        <string>actualiser</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="p3">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>320</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 3px;
border-top-color: rgb(255, 85, 0);

border-bottom-color: rgb(0, 170, 0);
border-right-color: rgb(255, 255, 0);
border-left-color: rgb(255, 255, 0);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_38">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>320</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>dessert</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>statistique</string>
      </attribute>
      <widget class="QComboBox" name="combo">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>180</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="verticalWidget" native="true">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>30</y>
         <width>311</width>
         <height>321</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="bbb"/>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>0</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-width: 4px;
border-color: red;</string>
     </property>
     <property name="text">
      <string>retour au menu</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
